README.md

```diff
@@ -1,45 +1,64 @@
 
-It is built as an extension to our [existing JS SDK](https://github.com/ably/ably-js), so you'll need an Ably JS client to be able to use this library. The usage instructions will help you set that up.
+## Development status
 
-### Glossary
+The Collaborative Spaces SDK is currently under development. 
 
-1. Space: A virtual collaborative space where members can collaborate in realtime with each other.
-2. Members: Users connected to the virtual collaborative space. The same user (identified by the clientId) can be present multiple times via different connections.
-3. Avatar Stack: A visual representation of the currently present and recently left list of members in the collaborative space. This is usually presented as a stack of profile pictures or initials along with other profile information such as name or role.
-4. (Coming soon) User Location: The live location of a member within the app. Depending on the nature of the app, this could be a slide number, a URL, a cell id, component id, etc.
-   1. Location Tracker: A dedicated component for tracking changes to a specific User Location
-5. (Coming soon) Live cursor: The pointer location of a member within the app.
+If you are interested in being an early adopter and providing feedback then you can [sign up](https://go.ably.com/spaces-early-access) for early access and are welcome to [provide us any feedback](https://go.ably.com/spaces-feedback).
 
-## Usage
+## Getting started
 
-### Pre-requisites
+Use the following snippets to quickly get up and running. More detailed [usage instructions](/docs/usage.md) are available, as are [class definitions](/docs/class-definitions.md) and information on [channel behavior](/docs/channel-behaviors.md).
 
-To use the Ably Spaces SDK, you'll need to have an Ably JS client set up and authenticated. To get started quickly, you can use [basic authentication](https://ably.com/docs/realtime/authentication#basic-authentication) which only requires an Ably API Key.
+You can also view a [live demo](https://spaces-demo.netlify.app/?space=nearby-lips-familiar). This uses an example slideshow to demonstrate updating an avatar stack and displaying user locations.
 
-1. [Sign up to a free Ably account](https://ably.com/signup)
-2. [Get your API Key](https://faqs.ably.com/setting-up-and-managing-api-keys) with publish, subscribe and presence capabilities enabled.
+### Prerequisites
 
-### Quickstart
+To begin, you will need the following:
 
-Create a new instance of the promise version of realtime client and pass that as a parameter to the Spaces constructor:
+* An Ably account. You can [sign up](https://ably.com/signup) for free.
+* An Ably API key. You can create API keys in an app within your [Ably account](https://ably.com/dashboard).
+  * The API key needs the following [capabilities](https://ably.com/docs/realtime/authentication#capabilities-explained): `publish`, `subscribe`, `presence` and `history`.
```

Comment:
A cropped screenshot of the dashboard will be useful here for new users.

Reply:
I added one in and then removed it as it doesn't add an awful lot of context, depending on what screen you use.

Reply:
Fair enough